src/views/SpaceSettings.vue

```diff
@@ -168,16 +168,16 @@ const {
   cancel: cancelDelete
 } = useConfirmDialog();
 
+const isViewOnly = computed(() => {
+  return !(isSpaceController.value || isSpaceAdmin.value);
+});
+
 onBeforeRouteLeave(async () => {
-  if (hasFormChanged.value) {
+  if (hasFormChanged.value && (isSpaceController.value || isSpaceAdmin.value)) {
```

Comment:
Why did you move `isViewOnly` up and then not use it?

Reply:
It was my intention to use it, but was not working for some reason

Reply:
Pushed something. Please check @wa0x6e